src/stories/components/RadioButton.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../css/radio-button.css';
+
+export const RadioButton = ({ groupName, inputValue, label, isDisabled }) => {
+    const disableLabel = isDisabled && "radio--label-disabled"
+    return(
+        <div>
+            <input type="radio" name={groupName} value={inputValue} disabled={isDisabled}/>
+            <label className={`radio--label ${disableLabel}`}>{label}</label>
+        </div>
+    )
+}
+
+RadioButton.propTypes = {
+    /*
+     * Radio button group
+     */
+    groupName: PropTypes.string,
+    /*
+     * Value of button
+     */
+    inputValue: PropTypes.string,
```

Comment:
Since it is a radio button the `inputValue` needs to be a boolean and not a string.

Reply:
Additionally, I'd just toggle the 'checked` prop of the input component for changing the "value" of the radio button. Take in another prop with the same name and add this as a control that can be toggled using the storybook UI. Right now, once I click on the button, there is no way for me to uncheck It 😭

Reply:
Isn't the value supposed to be like the returned value of the radio button itself? So like on submit the value sent to the server is the value. Otherwise the dev would have to loop through each button to figure out which one was selected.

Reply:
The input value of a radio button isn't returned, rather the "input value" returned would be of a group of radio buttons right? This component is meant to be a fractional radio button as far as I understand BUT that makes almost 0 sense to me so I'd say if you keep the "input value" prop then render at least 2 radio buttons in the group!